src/lib/evil/unposix/sys/types.h

```diff
@@ -1,21 +1,25 @@
 #ifndef UNPOSIX_SYS_TYPES_H
 #define UNPOSIX_SYS_TYPES_H
 
+#include_next <sys/types.h>
+
 #ifdef _MSC_VER
 #ifndef WIN32_LEAN_AND_MEAN
 # define WIN32_LEAN_AND_MEAN
 #endif
 
 #include <BaseTsd.h>
+#include <crtdefs.h>
+
 typedef SSIZE_T ssize_t;
+typedef _off_t off_t;
```

Comment:
`off_t` is defined at ucrt/sys/types.h

Reply:
O.o, docs about standard types doesn't say anything about it. - https://docs.microsoft.com/en-us/cpp/c-runtime-library/standard-types?view=vs-2019

Reply:
> O.o, *docs about standard types doesn't say anything about it*. - https://docs.microsoft.com/en-us/cpp/c-runtime-library/standard-types?view=vs-2019

Exactly my point. I had to read the Win10 SDK header to see it: https://github.com/tpn/winsdk-10/blob/9b69fd26ac0c7d0b83d378dba01080e93349c2ed/Include/10.0.16299.0/ucrt/sys/types.h

Reply:
wow....And other types need that `_CRT_DECLARE_NONSTDC_NAMES` too. Ok go on haah